lib/kdump_utils.pm

```diff
@@ -27,10 +27,16 @@ our @EXPORT = qw(install_kernel_debuginfo prepare_for_kdump
 
 sub install_kernel_debuginfo {
     zypper_call 'ref';
-    # JeOS uses kernel-default-base, except on aarch64 openSUSE
-    my $kernel    = script_output('rpm -qf --qf %{name} /boot/initrd-$(uname -r)');
-    my $debuginfo = script_output('rpmquery --queryformat="%{NAME}-%{VERSION}-%{RELEASE}\n" ' . $kernel . '| sort --version-sort | tail -n 1');
-    $debuginfo =~ s/$kernel/$kernel-debuginfo/g;
+    my $kernel = script_output('rpm -qf --qf %{name} /boot/initrd-$(uname -r)');
+    my $debuginfo;
+    # on jeos there is no 'kernel-default-base-debuginfo', use 'kernel-default-debuginfo' instead.
+    if (is_tumbleweed && is_jeos && ($kernel eq 'kernel-default-base')) {
```

Comment:
`is_tumbleweed` is the wrong condition, it also appears on SLE 15 SP2+ and Leap 15.2+.